examples/webgl_loader_mdd.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js WebGL - MDDLoader</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<link type="text/css" rel="stylesheet" href="main.css">
+	</head>
+	<body>
+
+		<div id="info">
+			<a href="https://threejs.org" target="_blank" rel="noopener">three.js</a> - MDDLoader
+		</div>
+
+		<script type="module">
+
+			import * as THREE from '../build/three.module.js';
+
+			import { MDDLoader } from './jsm/loaders/MDDLoader.js';
+
+			var camera, scene, renderer, mixer, clock;
+
+			init();
+
+			function init() {
+
+				scene = new THREE.Scene();
+
+				camera = new THREE.PerspectiveCamera( 35, window.innerWidth / window.innerHeight, 0.1, 100 );
+				camera.position.set( 10, 10, 10 );
+				camera.lookAt( scene.position );
+
+				clock = new THREE.Clock();
+
+				//
+
+				var hemiLight = new THREE.HemisphereLight( 0xffffff, 0x444444 );
+				hemiLight.position.set( 0, 20, 0 );
+				scene.add( hemiLight );
+
+				var dirLight = new THREE.DirectionalLight( 0xffffff );
+				dirLight.position.set( - 3, 10, - 10 );
+				scene.add( dirLight );
+
+				//
+
+				var loader = new MDDLoader();
+				loader.load( 'models/mdd/cube.mdd', function ( result ) {
+
+					const morphTargets = result.morphTargets;
+					const clip = result.clip;
+					// clip.optimize(); // optional
+
+					var geometry = new THREE.BoxBufferGeometry();
```

Comment:
Do I understand it correctly that a MDD file has no information of how points connect to each other?

Reply:
Yes, the MDD files has to exactly match the used geometry. Meaning even the order of vertices must be equal.

Reply:
Got it. Thanks!